### PR TITLE
Move international sites (Indonesia) to Docker and add Pantheon integration

### PR DESCRIPTION
**Pull request by crittermike**
#### What's this PR do?
This PR does a few things:

- Switches to Docksal (Docker-based) for local environments
- Has new scripting to build the codebase locally and clone the DB
- Includes a script to push the current codebase to Pantheon's dev environment

#### How should this be reviewed?
1. [Install Docksal](http://docksal.readthedocs.io/en/master/getting-started/env-setup-native/#using-docksal-with-docker-for-macwindows-new-users)
2. Check out this feature branch
3. Run `fin init` and make sure nothing breaks and that at the end you have a working local site.

### CODE REVIEW

File: .docksal/commands/init [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+NC='\033[0m'
+
+cd "$PROJECT_ROOT"
+
+echo
+echo -e "${yellow}Starting the containers...${NC}"
+fin project start
+
+${PROJECT_ROOT}/.docksal/commands/rebuild
+fin exec ${PROJECT_ROOT}/.docksal/commands/db-import
```
Comment (mirie) [inline .docksal/commands/init]: I was getting the following error: `/bin/bash: /Users/mirie/Sites/phoenix/.docksal/commands/db-import: No such file or directory`. I took out `fin exec` and there was no longer the same complaint (a new complaint appeared). 
I'm guessing it's needed for the db creds? After I removed `fin exec`, the script ran until db import and then failed on: 
```
Importing database dump...
mysql: [Warning] Using a password on the command line interface can be insecure.
ERROR 2005 (HY000): Unknown MySQL server host 'db' (0)
```

**Review by mirie — CHANGES_REQUESTED** [review]
Let's add to the instructions the Terminus installation requirement to the README.md. The db-import script won't work without it. I'm not sure what to do about the required machine token however. I went ahead and created a new Machine Token: `Mai Irie Terminus Test Token`. Looks like I can only authenticate with Matt’s email even though I added my email as a team member to the site (I guess this email/token thing is at the account level). I might be misunderstanding how the terminus authentication is supposed to work. Is the idea to share the token?  

When re-running `fin init`, Is there a way not be asked to `Enter passphrase for /root/.ssh/id_rsa` each time?

File: .docksal/commands/init (hunk 1 above)
Comment (mirie) [reply .docksal/commands/init]: ok, did some more fiddling, looks like if I remove `${PROJECT_ROOT}/` from line 15 everything executes OK.

Comment (mirie) [reply .docksal/commands/init]: hmmm, despite the Importing database step appearing to complete, I'm still taken to the http://dosomething.docksal/install.php page. Looking https://docksal.readthedocs.io/en/master/advanced/drupal-settings/ it appears that the settings.php file needs to be updated as part of the init process, or should be noted if manual changes are needed in a local.settings.php file

Comment (crittermike) [reply .docksal/commands/init]: @mirie re: the settings file, this line is supposed to deal with that: https://github.com/DoSomething/phoenix/pull/7549/files/99fc30f795989e8e689b92a218d4f6c3fdbc7cf1#diff-9258306d1c5abc5a09607d506e182f15R17

Did that line not get added to the end of the settings.php file for you?

Comment (mirie) [reply .docksal/commands/init]: Discussed over Slack: was missing the local.settings.php file - addressed in https://github.com/DoSomething/phoenix/pull/7549/commits/d6ba56985d17c9f2f6cc9ba3ae37b3b831a027c3

**Comment by mirie** [comment]
Hi @mikecrittenden Is this ready for review again? Should I do another test?

**Comment by crittermike** [comment]
@mirie not yet, still need to figure out the issues you ran into with the Machine Tokens. I thought that adding it here (https://github.com/DoSomething/phoenix/pull/7549/files#diff-744c7fdbc06871568a97c591cf35d773R31) would have worked for everyone but apparently not.

**Comment by mirie** [comment]
Discussed over slack: @mikecrittenden to take a quick timeboxed peek (hadn't had time due to other commitments) about token issue. Worst case is adding output explaining how to authenticate. Then I'll re-test when I get the go ahead and we'll move onto the Mandrill issue.

**Review by mirie — APPROVED** [review]
Re-ran, looks great!

**Pull request merged**